src/main/java/net/sf/jabref/gui/errorconsole/ErrorConsoleView.java

```diff
@@ -80,14 +90,91 @@ private void copyLogButton() {
+        graphicColumn.setFillWidth(false);
+        graphicColumn.setHgrow(Priority.NEVER);
+        ColumnConstraints textColumn = new ColumnConstraints();
+        textColumn.setFillWidth(true);
+        textColumn.setHgrow(Priority.ALWAYS);
+        headerGrid.getColumnConstraints().setAll(graphicColumn, textColumn);
+        headerGrid.setPadding(new Insets(10));
+
+        Image image = new Image("https://cdn2.iconfinder.com/data/icons/windows-8-metro-style/512/console.png");
+        ImageView imageView = new ImageView(image);
+        imageView.setFitWidth(48);
+        imageView.setFitHeight(48);
+        StackPane stackPane = new StackPane(imageView);
+        stackPane.setAlignment(Pos.CENTER);
+        headerGrid.add(stackPane, 0, 0);
+
+        Label headerLabel = new Label(Localization.lang("We now give you an insight into the inner workings of JabRef's brain. ") +
+                Localization.lang("This might help to diagonalize the root of problems. ") + System.lineSeparator() +
+                Localization.lang("Please use the button below to inform the developers about an issue."));
+        headerLabel.setWrapText(true);
+        headerLabel.setPadding(new Insets(10));
+        headerLabel.setAlignment(Pos.CENTER_LEFT);
+        headerLabel.setMaxWidth(Double.MAX_VALUE);
+        headerLabel.setMaxHeight(Double.MAX_VALUE);
+        headerGrid.add(headerLabel, 1, 0);
+
+        return headerGrid;
+    }
+
+    //style the list view with icon and message color
+    private void listViewStyle() {
+        // handler for listCell appearance (example for exception Cell)
+        allMessage.setCellFactory(new Callback<ListView<ObservableMessageWithPriority>, ListCell<ObservableMessageWithPriority>>() {
+            @Override
+            public ListCell<ObservableMessageWithPriority> call(ListView<ObservableMessageWithPriority> listView) {
+                return new ListCell<ObservableMessageWithPriority>() {
+                    @Override
+                    public void updateItem(ObservableMessageWithPriority omp, boolean empty) {
+                        super.updateItem(omp, empty);
+                        if (omp != null) {
+                            setText(omp.getMessage());
+                            getStyleClass().clear();
+                            if (omp.getPriority() == MessagePriority.HIGH) {
+                                setGraphic(listImageResize("http://www.iconsdb.com/icons/preview/red/info-xxl.png"));
+                                getStyleClass().add("exception");
+                            } else if (omp.getPriority() == MessagePriority.MEDIUM) {
+                                setGraphic(listImageResize("http://www.iconsdb.com/icons/preview/black/info-xxl.png"));
+                                getStyleClass().add("output");
+                            } else {
+                                setGraphic(listImageResize("http://www.iconsdb.com/icons/preview/royal-blue/info-xxl.png"));
```

Comment:
Since you already set style classes to the rows and specify their style in the stylesheet, maybe you can also set the icon there?